src/common.cpp

```diff
@@ -1873,22 +1880,18 @@ std::string get_executable_path(const char *argv0) {
     } else {
         return std::string(buff);
     }
-#else
-    // On other unixes, fall back to the Linux-ish /proc/ directory
+#elif (defined(__linux__) || defined(sun) || defined(__sun))
```

Comment:
This is unnecessary enumeration, the `#else` can stay.

It's *fine* to try /proc, and if that doesn't work try argv0.